### ISSUE TITLE
Dismisses keyboard when logging

### ISSUE THREAD

**Issue by wgrand** [tadija/AEConsole#4]
Whenever I log something, this line is called and it removes focus from any UITextFields and thus dismisses the keyboard:

https://github.com/tadija/AEConsole/blob/1f9a7ac78a185d8f53e2e4867b82322a880d417a/Sources/AEConsole.swift#L91

**Comment by penghui** [comment]
I ended up with remove that line of code and added a UIWindow extention to handle shake gesture:
```Swift
extension UIWindow {
    override open func motionEnded(_ motion: UIEventSubtype, with event: UIEvent?) {
        if motion == .motionShake {
            AEConsole.toggle()
        }
    }
}
```

**Comment by tadija** [comment]
Hi @wgrand and @penghui, thanks for reporting this bug, I think it should be solved now in 0.5.0.
